section2/reflection.md

```diff
@@ -2,14 +2,34 @@
 
 1. What are two points from the Growth Mindset article and/or video that either resonated with you, or were brand new to you?
 
+- In Kristin Villanuava's blog part 1, she discusses using the Zone of Proximal Development method. This makes a lot of sense when it comes to effectively and efficiently developing a skill set.
+
+- In part 2, she discusses that understanding that there is always room to grow will give you the mindset essential for becoming an effective programmer.
+
 2. In which ways do you currently demonstrate a Growth Mindset? In which ways do you _not_?
 
+I would say I'm a very eager learner and enjoy practicing as well as not being a quitter. In my youth, I was more comfortable with giving up on something but as an adult my eagerness to learn has helped me persevere and not give up on tasks.
+
 3. In this section, we removed some of the supports that you had in Section 1. We didn't give the directions for how to run a file in node and made some directions a bit more vague, intentionally. How did that feel? What was uncomfortable about it? How did it support your learning?
 
+It felt fine. I did have to go back and look at the node file path from section 1 to remember it exactly but I doubt I will have to do that anymore. I liked the more vague directions. It challenged me, which is what I wanted so that I could see where I was at.
+
 4. What is a conditional statement? Give one example of a daily life conditional. Give one example of where a conditional is probably used in a web application you use.
 
+A conditional statement is when a "condition" must be met in order for a piece of code to execute. If the condition isn't met, the next condition is evaluated to determine if a condition is met and so on.
+
+An example in daily life: If you don't renew your driver's license by a specified date, you will not legally be permitted to drive.
+
+Rosetta Stone language learning uses conditionals to determine if you have grasped enough of the lesson to move onto the next lesson or you must go over the lesson again.
+
 5. How do you add multiple conditions to an `if` statement? In your own words, explain how to program reads them and determines what to do.
 
+the first condition is read as "if 'condition' is met," code will execute, "else if," test next condition and so on.
+
 6. What tools are available to you, as a developer, if you want to check that TWO conditions are met? Or, if you want to check that 1 of 2 conditions are met, before running a block of code?
 
-7. What questions do you still have about `if` statements and/or functions?
\ No newline at end of file
+logical operators such as && to determine if "condition" and "condition" are met and || to determine if "condition" or "condition" are met.
+
+7. What questions do you still have about `if` statements and/or functions?
+
+I mostly get it but I'm not 100% on when to end a conditional with just "else."
```

Comment:
For any specific situations that you want to cover, you will usually have an `if` or `else if`, `else` is usually used as to catch any other conditions that are not accounted for.

Reply:
Gotcha. It kind of seemed like this but I guess I wanted someone to exclusively say that. Thanks!